src/components/Garden/Feed/WrapToken.js

```diff
@@ -19,27 +19,13 @@ import wrappedIcon from '@assets/wrappedIcon.svg'
 import unwrappedIcon from '@assets/unwrappedIcon.svg'
 import claimRewardsIcon from '@assets/rewardsWrapperIcon.svg'
 
-const modeAttributes = {
-  wrap: { icon: unwrappedIcon, button: { mode: 'strong', label: 'Wrap' } },
-  unwrap: {
-    icon: wrappedIcon,
-    button: { mode: 'strong', label: 'Unwrap' },
-    hint:
-      'This amount can be used to vote on proposals. It can be unwrapped at any time.',
-  },
-  claim: {
-    button: { mode: 'normal', label: 'Claim' },
-    icon: claimRewardsIcon,
-  },
-}
-
 function WrapToken({ onClaimRewards, onUnwrapToken, onWrapToken }) {
   const { token, wrappableToken } = useGardenState()
 
   const loading =
     token.accountBalance.eq(-1) || wrappableToken.accountBalance.eq(-1)
 
-  const [earnedRewards, rewardsLink] = useUnipoolRewards()
+  const [earnedRewards, rewardsLink, rewardAPY] = useUnipoolRewards()
```

Comment:
Now yes let's do the  formatting `const rewardAPYFormatted = (rewardAPY && !rewardAPY.eq(0)) ? rewardAPY.multipliedBy(100).toFixed(2) : null`

And also let's decide before hand which icon we want to use so we can do something like : 
`const unwrappedImage = (rewardAPY && !rewardAPY.eq(0)) ? apyIcon : unwrappedIcon`